google-cloud-pub-sub-grpc/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/grpc/javadsl/GooglePubSub.scala

```diff
@@ -68,6 +68,34 @@ object GooglePubSub {
       .mapMaterializedValue(japiFunction(flattenCs))
       .mapMaterializedValue(japiFunction(_.toCompletableFuture))
 
+  /**
+   * Create a source that emits messages for a given subscription using a synchronous PullRequest.
+   *
+   * The materialized value can be used to cancel the source.
+   *
+   * @param request the subscription FQRS field is mandatory for the request
+   * @param pollInterval time between PullRequest messages are being sent
+   */
+  def subscribePolling(
+      request: PullRequest,
+      pollInterval: Duration
+  ): Source[ReceivedMessage, CompletableFuture[Cancellable]] =
+    Source
+      .setup { (mat, attr) =>
+        val cancellable = new CompletableFuture[Cancellable]()
+
+        val client = subscriber(mat, attr).client
+
+        Source
+          .tick(pollInterval, pollInterval, request)
```

Comment:
```suggestion
          .tick(0.seconds, pollInterval, request)
```

Can you please fix this in `subscribe` as well.